src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_larget_disk.yaml

```diff
@@ -0,0 +1,1044 @@
+interactions:
+- request:
```

Comment:
We will need to delete this file after renaming the test. Should be: test_vm_large_disk.yaml